.travis.yml

```diff
@@ -81,6 +81,7 @@ after_script:
 jobs:
   include:
     - stage: lint
+      env:
       sudo: false
       cache: false
       addons:
```

Comment:
As you touch this file, might as well do some other cleanup and e.g. remove the apt addon stage completely. The installation is not needed (and only consumes time). See diff, that I confirmed to be working:

```diff
diff --git a/.travis.yml b/.travis.yml
index ec78896f44..c03a2bea24 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -82,13 +82,10 @@ jobs:
     - stage: lint
       sudo: false
       cache: false
-      addons:
-        apt:
-          packages:
-            - python3-pip
-            - shellcheck
+      language: python
+      python: '3.6'
       install:
-        - travis_retry pip3 install flake8 --user
+        - travis_retry pip install flake8
       before_script:
         - git fetch --unshallow
       script:
```

Reply:
Nice, 26 seconds saved: [before](https://travis-ci.org/bitcoin/bitcoin/builds/406455557?utm_source=github_status&utm_medium=notification) [after](https://travis-ci.org/bitcoin/bitcoin/builds/406632797?utm_source=github_status&utm_medium=notification)